packages/sil-governance/src/routes/dashboard/components/Proposal/CollapsingData/DisplayOptions/SetValidators.tsx

```diff
@@ -0,0 +1,30 @@
+import { SetValidatorsAction } from "@iov/bns";
+import { Block, Typography } from "medulas-react-components";
+import React from "react";
+
+interface Props {
+  readonly action: SetValidatorsAction;
+}
+
+const SetValidators = ({ action }: Props): JSX.Element => {
+  let validators = [];
+
+  for (const validator in action.validatorUpdates) {
```

Comment:
`for...in` can be dangerous. Prefer `for (const [key, value] of Object.entries(action.validatorUpdates))` or better yet `const validators = Object.entries(...).map(...)`.

Does your linter not complain about this?

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in#Why_Use_for...in